CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 ### Changed
 
 - ⚠️ `gtfs-importer`: upgraded [`postgis-gtfs-importer`](https://github.com/mobidata-bw/postgis-gtfs-importer) to [`v5-2025-02-19T00.49.01-20c1b09`](https://github.com/mobidata-bw/postgis-gtfs-importer/tree/20c1b09) – This is a breaking change because future GTFS imports won't work without a manual migration step. Please run `docker compose --env-file .env --env-file .env.local exec gtfs-db /bin/sh -c 'env PGUSER="$POSTGRES_USER" PGPASSWORD="$POSTGRES_PASSWORD" psql gtfs_importer -1 -c "CREATE TABLE IF NOT EXISTS latest_successful_imports (db_name TEXT PRIMARY KEY, imported_at INTEGER NOT NULL, feed_digest TEXT NOT NULL); INSERT INTO latest_successful_imports (db_name, imported_at, feed_digest) SELECT db_name, split_part(db_name, '\''_'\'', 2)::integer AS imported_at, split_part(db_name, '\''_'\'', 3) AS feed_digest FROM latest_import"'` to adapt the `gtfs_importer` database to the new way `postgis-gtfs-importer` keeps track of the imports (a table `latest_successful_imports`).
+- `geoserver`: switch to [a custom Docker image](https://github.com/mobidata-bw/ipl-geoserver/pkgs/container/ipl-geoserver) that already includes the vector tiles plugin, so that the container doesn't have to download the plugin at runtime anymore.
```

Comment:
Only the vector tiles plugin or the inspire plugin as well? It was removed temporailly due to download issues, but is intended to be included